training_presets/#pixart alpha 1.0.json

```diff
@@ -1,12 +1,12 @@
 {
-    "base_model_name": "PixArt-alpha/PixArt-XL-2-1024-MS",
```

Comment:
Python code is indented with 4 spaces, can you configure the same for json files? Also, please line sorting in json files. It doesn't make sense to sort all json files alphabetically